src/tools/brush.lua

```diff
@@ -43,18 +59,87 @@ function tool.mouseclicked(x, y, button, istouch, pressed)
     local cloneButton = configs.editor.objectCloneButton
 
     if button == actionButton then
-        brushToolUtils.handleActionClick(tool, x, y)
-
+        if tool.mode == "pencil" then
+            brushToolUtils.handleActionClick(tool, x, y)
+        elseif tool.mode == "dither" then
+            local room = state.getSelectedRoom()
+
+            if room then
+                local px, py = viewportHandler.getRoomCoordindates(room, x, y)
+                local tx, ty = viewportHandler.pixelToTileCoordinates(px, py)
+
+                brushToolUtils.startTileSnapshot(tool)
+                brushHelper.placeTile(room, tx + 1, ty + 1, tool.material, tool.layer)
+                brushHelper.placeTile(room, tx + 2, ty + 2, tool.material, tool.layer)
```

Comment:
when i tested this, the second tile was placed but didn't display until re-autotiled; i have observed this with other tools so i think it's a display bug